Core/include/Acts/EventData/CompositeSpacePointCalibrator.hpp

```diff
@@ -0,0 +1,37 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/CompositeSpacePoint.hpp"
+#include "Acts/Utilities/CalibrationContext.hpp"
+namespace Acts::Experimental {
+
+/// @brief Interface concept to calibrate CompositeSpacePoints
+
+template <typename Calibrator_t, typename SpacePoint_t>
+concept CompositeSpacePointFastCalibrator =
+    CompositeSpacePoint<SpacePoint_t> &&
+    requires(const Calibrator_t calibrator, const Acts::CalibrationContext& ctx,
+             const SpacePoint_t& spacePoint, const double t0) {
+      /// @brief Returns the drift velocity of the straw measurement's radius - time relation
+      ///        which is defined as the first derivative of the relation.
+      /// @param ctx: Calibration context to access the calibration constants (Experiment specific)
+      /// @param spacePoint: Reference to the calibrated space point
```

Comment:
```suggestion
      /// @param spacePoint: Reference to the calibrated space point
      /// @param t0: Time of the hard scatter in the detector frame
```
not sure if this is true - just a suggestion